renovate.json

```diff
@@ -1,4 +1,8 @@
 {
+  "assignees": [
+    "mrkvon",
```

Comment:
@mrkvon lemme know if you'd prefer not to get pings from dependency upgrade PRs. :-)

I think at this point both you and me know the repository best and can merge these fairly fast.

Reply:
I think I'm fine with trying it out. Can always remove later if i get bothered by that. :slightly_smiling_face:

Reply:
Opened #1221 removing myself. :slightly_smiling_face: I'm still open to reviewing, just the notifications get overwhelming.